tests/performance-test/deploy/performance-test-tg.sh

```diff
@@ -1,21 +1,93 @@
 #!/bin/bash
 # Sets up the envirovnment for performance testing
+
+#set -e
+
+oc project openshift-monitoring
+OCP_PROM_HOST=$(oc get routes --field-selector metadata.name=prometheus-k8s -o jsonpath="{.items[0].spec.host}")
+OCP_DATASOURCE=$(oc get secret -n openshift-monitoring grafana-datasources -o jsonpath='{.data.prometheus\.yaml}' \
+                | base64 -d)
+OCP_DATASOURCE=$(echo $OCP_DATASOURCE | python -c "import sys, json; source=json.load(sys.stdin)['datasources'][0]; \
+                source['url']='https://$OCP_PROM_HOST'; source['name']='OCPPrometheus'; print(json.dumps(source))")
+
+oc project sa-telemetry
+GRAF_HOST=$(oc get routes --field-selector metadata.name=grafana -o jsonpath="{.items[0].spec.host}")
+SAF_PROM_HOST=$(oc get routes --field-selector metadata.name=prometheus -o jsonpath="{.items[0].spec.host}")
+
+SAF_DATASOURCE=$(cat << EOF
+    {
+        "name"  :   "SAFPrometheus",
+        "type"  :   "prometheus",
+        "url"   :   "http://$SAF_PROM_HOST",
+        "access":   "direct",
+        "basicAuth" :false,
+        "jsonData": {
+            "timeInterval"  :   "1s"
```

Comment:
Seems like it might be worth to just paramaterize this?

Reply:
For collectd-tg yes. In fact, most of this script's functionality is now apart of the performance-test.sh script and it should be stripped down. Priority shifted towards getting everything working with telemetry-bench first, therefore a lot of the collectd-tg is midway through re factoring and probably does not work at all anyway. Really, tg stuff is outside the scope of this feature request and should be dealt with separately